app/code/community/Vindi/Subscription/Helper/Order.php

```diff
@@ -51,7 +51,7 @@ public function getOrderFromVindi($billId)
 		if (! $bill) {
 			return false;
```

Comment:
Aqui como a gente não usa mais todas as outras verificações, não pode convir retornar `NULL`?
```suggestion
			return NULL;
```